source/javascripts/core/api/StepApi.ts

```diff
@@ -310,6 +310,7 @@ async function getAlgoliaStepInputsByCvs(cvs: string): Promise<StepInputVariable
 export { AlgoliaStepResponse, AlgoliaStepInputResponse, StepInfo, StepApiResult };
 
 export default {
+  getAlgoliaClients,
   getAlgoliaSteps,
```

Comment:
I would simply rename it, since Algolia is an implementation detail.
```suggestion
  getSteps,
```